specs/Spec_for_editing_user.adoc

```diff
@@ -0,0 +1,170 @@
+# User Management: Editing User view for Admin
+
+## Problem description
+
+This spec talks about viewing or editing information stored about a single user.
+
+## Use Cases
+
+The flow goes as follows:-
+
+* A user logs in as Admin
+* Clicks on Admin tab in Navigation
+* The navigation drops down
+* Clicks on *Users*
+* Clicks on *Edit* option for a particular user
+
+User will land onto https://redhat.invisionapp.com/share/KNB25OEQT#/screens/226063805[Edit User] view page
+
+## Proposed workflow
+
+### UI
+
+Edit user view consist of a form:-
+
+* ```User ID``` - Not Editable
+* ```Type : Internal``` - Not Editable
```

Comment:
What is type internal? Is it like `admin` is internal user and cannot be edited?